### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by jgresham** [argentlabs/userpage#33] [Ideas]
Is there a roadmap for this page? Willing to help out on some features or ideas.

**Comment by janek26** [comment]
Hey! Thanks for the interest :-)

tbh there are no concrete next steps planned right now. This page is mainly used as a L1->zksync bridge by our users. If you're missing something please feel free to open a PR and I'll be happy to support

Personally I think it would be a very cool feature if you would be able to bridge multiple assets from your wallet to zksync. If you do so from another Argent Vault (L1) wallet, it would even be possible to approve and bridge multiple tokens all in one transaction, saving time and costs, using the multi call method build into argent wallets as documented in https://docs.argent.xyz/wallet-connect-and-argent#multicall

**Reply by jgresham** [reply]
Thinking about taking this up soon. Please let me know if it isn't relevant or if I am missing anything here.

1. On `/send` route, I'd change the token single select into an optional multi-select
2. Allow only zkSync supported erc20s (https://zkscan.io/explorer/tokens/).
3. Include all the selected tokens' `approve` and `depositERC20` objects in the `wc_multiCall` in `send.ts`.

I'm having trouble finding a good Ropsten faucet for Dai or any of the zkSync supported erc20s, if you could point me in the right direction for that. Thanks!

**Reply by jgresham** [reply]
Next iteration could be including transferring multiple NFTs (https://zksync.io/api/sdk/js/nfts.html#transfer-nft) and maybe a button to just transfer everything :)

**Comment by jgresham** [comment]
@janek26 thanks for the reply.

That'd be nice for users considering gas fees. Ability to click once to move everything possible would be ideal too. I'll take a look at it!
